repo_cleanups/remove_stale_rpm.sh

```diff
@@ -0,0 +1,231 @@
+#!/bin/bash 
```

Comment:
It looks like you forgot to call `git rm` here too. For the future, `git mv` is a useful command to rename files with git being aware of it. `git mv foo bar` is basically the same as:

```
mv foo bar
git add bar
git rm foo
```